pyproject.toml

```diff
@@ -40,7 +40,7 @@ werkzeug = "2.0.3" # Flask breaks with later versions of Werkzeug
 filelock = "^3.0.12"
 mypy = "^0.790"
 pytest = "^6.2.4"
-pytest-cov = "^2.12.1"
+pytest-cov = "^3.0.0"
```

Comment:
Not sure if there were any concerns with this upgrade but seemed worthwhile if the older version has been blocking development for recent spin ups.